src/Qir/Execution/Tools/Driver/QirFullStateSimulatorInitializer.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation.
```

Comment:
What is this initializer concept? What was this moved from?

Reply:
Chatted offline. In summary, this was not moved, it's a new concept to enable creating drivers for multiple simulators and not only the full state one.